### PR TITLE
Addressed Justin's 16, 17, 18

### PR DESCRIPTION
**Pull request by vbuterin**

### CODE REVIEW

File: specs/core/0_beacon-chain.md [hunk 1]
```diff
@@ -597,7 +597,9 @@ def deposit(deposit_params: bytes[2048]):
         self.receipt_tree[index] = sha3(concat(self.receipt_tree[index * 2], self.receipt_tree[index * 2 + 1]))
     self.total_deposit_count += 1
     if self.total_deposit_count == 16384:
-        log.ChainStart(self.receipt_tree[1], timestamp_bytes8)
+        timestamp_day_boundary = (block.timestamp - block.timestamp % 86400) + 86400
```
Comment (djrtwo) [inline specs/core/0_beacon-chain.md]: This needs a type def
```suggestion
        timestamp_day_boundary: timestamp = (block.timestamp - block.timestamp % 86400) + 86400
```

File: specs/core/0_beacon-chain.md [hunk 2]
```diff
@@ -597,7 +597,9 @@ def deposit(deposit_params: bytes[2048]):
...
+        timestamp_day_boundary_bytes8: bytes[8] = slice(convert(timestamp_day_boundary, 'bytes32'), 24, 8)
+        log.ChainStart(self.receipt_tree[1], timestamp_day_boundary_bytes8 )
```
Comment (djrtwo) [inline specs/core/0_beacon-chain.md]: remove extra space before end parenthesis 
```suggestion
        log.ChainStart(self.receipt_tree[1], timestamp_day_boundary_bytes8)
```

**Comment by JustinDrake** [comment]
LGTM. `SECONDS_IN_A_DAY := 86400`?

**Comment by vbuterin** [comment]
> LGTM. SECONDS_IN_A_DAY := 86400?

Just added. Also went through and made sure the code actually compiles in Vyper.

**Comment by hwwhww** [comment]
@vbuterin ready to merge?

**Pull request merged**